cdap-ui/app/cdap/components/Home/Welcome/index.tsx

```diff
@@ -40,7 +40,7 @@ interface IWelcomeState {
   showAgain: boolean;
 }
 
-export default class Welcome extends React.PureComponent<void, IWelcomeState> {
+export default class Welcome extends React.PureComponent<{}, IWelcomeState> {
```

Comment:
Just curious why is this change needed?

Reply:
void is not allowed for Props type apparently. It was throwing tslint error.